tests/federation/transport/test_knocking.py

```diff
@@ -0,0 +1,303 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we got rid of all of these `utf-8` lines in other code...we should probably re-run pyupgrade after this is merged (or maybe before).

Reply:
I don't think we've run it across the whole codebase yet. Running `pyup_dirs --py36-plus --keep-percent-format synapse/ tests/` (`pyup_dirs` is a helper package that can recursively pyupgrade check files) produces many, many changes.

I can easily remove the `utf-8` lines from this PR though.

Reply:
`grep -r "coding: utf-8" synapse tests` produces a single hit for me in `synapse/config/account_validity.py`.

Please remove it.